### ISSUE TITLE
Have a web base GUI for monitoring and easy remote access (Enhancement)

### ISSUE THREAD

**Issue by Wowowe** [cTn-dev/Excavator-GUI#15]
I'm suggesting an enhancement to the Gui where in we can directly access the GUI through web, so that when we want to access and check the gui.. we can just paste the url to any web as long as we are connected to the LAN and directly see the GUI itself. I know excavator already it built-in http://127.0.0.1:38081/  <-- but its limited it would be nice if the GUI itself would be the one we can access directly. So we wont be using anymore rdp or VNC to access the rig itself just to monitor and do some changes.  I hope you get what I mean.

Like for example we are within the LAN we just access http://127.0.0.1:4000/  (just example) <-- when we access that url we would see the "Excavator GUI by cTnko" but its a web base form in there we can do some changes same as the GUI itself. The purpose of this one is for easy remote access and monitoring, so we would know if the excavator is still running real-time. 

More power to cTnko!
It just keeps getting better day by day! POWER!

**Comment by cTn-dev** [comment]
Hi, i actually started working on a backend for this a while back.
Currently the development is a bit on hold (i am waiting to see if the adoption for excavator or excavator GUI increases).
Since the miner is great and supports quite a few algos, sadly it isn't the fastest in any of them, which puts people off (and i understand that), the performance is more important than the features for miners.

**Comment by Wowowe** [comment]
Yes I have to agree on that excavator really need to work on their miner on making it more faster than other miners! I think the latest Nicehash miner is using the latest excavator!
